awstesting/assert.go

```diff
@@ -1,9 +1,11 @@
 package awstesting
 
 import (
+	"bytes"
 	"encoding/json"
 	"encoding/xml"
 	"fmt"
+	"github.com/aws/aws-sdk-go/private/protocol/xml/xmlutil"
```

Comment:
nit, using `goimports` will correctly order the pkg imports so stdlib are separate from third party.